### PR TITLE
Error handling for DB connection

### PR DESCRIPTION
**Pull request by spiray**
I Implemented error handling for the DB connection. Let me know if this is a good method.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -7,7 +7,11 @@ const { server: { port, host } } = require('./config');
 require('dotenv').config();
 
 // Database connection
-require('./helpers/databaseConnection');
+require('./helpers/databaseConnection')
+  .then(
```
Comment (joshuaferr1s) [inline src/index.js]: This .then chain would look a lot nicer on the helpers export rather than on a require.

**Review by joshuaferr1s — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (spiray) [reply src/index.js]: I agree! I did not know you were able to do that. Thanks

**Review by joshuaferr1s — APPROVED** [review]
Looks good. Thanks @spiray

**Comment by spiray** [comment]
[Handle connection errors for DB](https://trello.com/c/DWEWTCUJ/83-handle-connection-errors-for-db)

**Review by rabbitwerks — APPROVED** [review]

**Pull request merged**